minargon/static/conf/Sy1527Ech.sub.fnal

```diff
@@ -0,0 +1,196 @@
+file db/Sy1527ch.tpl{ 

```

Comment:
Is this information that can't be accessed from a database? Does it need to be in a file like this.

Reply:
Yes, this is to future-proof the page. This is a mapping of power supply channel to PMT.
The mapping may change in the future, so in order to avoid changing the mapping on the website, the new version of this file can be re-uploaded to replace this file and the mapping on the page will be updated automatically